rosdep/python.yaml

```diff
@@ -9251,6 +9251,12 @@ python3-smbus:
     '*': [python3-i2c-tools]
     '8': null
   ubuntu: [python3-smbus]
+python3-smbus2:
+  debian: [python3-smbus2]
+  ubuntu:
+    '*': [python3-smbus2]
+    focal: null
+    jammy: null
```

Comment:
There actually is a Jammy package so unless there's something known to be wrong with it, we don't need to null this key: https://packages.ubuntu.com/jammy/python3-smbus2
```suggestion
```